Tests/MoyaProviderSpec.swift

```diff
@@ -771,6 +771,48 @@ class MoyaProviderSpec: QuickSpec {
 
         }
 
+        describe("a provider with upload progress tracking") {
+            var provider: MoyaProvider<HTTPBin>!
+            beforeEach {
+                provider = MoyaProvider<HTTPBin>()
+            }
+
+            it("tracks progress of request") {
+
+                let url = Bundle(for: MoyaProviderSpec.self).urlForImageResource("testImage.png")!
```

Comment:
Hmm, CI is failing on this one `Bundle.urlForImageResource("testImage.png")!`, could you check if it compiles for you locally?

Reply:
I believe this is failing because I don't think`urlForImageResource` is available on iOS.

Reply:
yes I test with first target only, mac os, my bad
fixed